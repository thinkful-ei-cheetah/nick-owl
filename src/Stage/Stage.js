import React from 'react'

export default function Stage() {
  return (
    <section className="stage">
      
    </section>
  )
}
