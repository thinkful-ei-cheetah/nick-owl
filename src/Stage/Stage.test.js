import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage';
import renderer from 'react-test-renderer';

describe('Stage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Stage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders UI as expected', () => {
    const tree = renderer
      .create(<Stage />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});