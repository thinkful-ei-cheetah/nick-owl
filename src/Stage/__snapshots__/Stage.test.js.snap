// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage Component renders UI as expected 1`] = `
<section
  className="stage"
/>
`;
